src/Components/ImagePreloader.ts

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Light from "../Images/Light.jpg";
+
+const ImagePreloader = () => {
```

Comment:
What is the purpose of this component?

Reply:
The ImagePreloader.ts component loads the image from the Light theme as soon as the app is loaded (it is rendered in App.tsx). It does not display anything on the page, but because the image is already downloaded, when the user changes the theme to Light, it changes instantly.
If we do not load this preloader, when the user changes the theme for the first time, the Dark background image disappears and a white background is displayed until the Light theme is downloaded